src/ceph-volume/ceph_volume/util/device.py

```diff
@@ -105,6 +108,22 @@ def is_device(self):
             return self.disk_api['TYPE'] == 'device'
         return False
 
+    @property
+    def is_valid(self):
+        def reject_device(item, value, reason):
+            if item in self.sys_api:
+                if self.sys_api[item] == value:
+                    self._rejected_reasons.append(reason)
+        reject_device('removable', 1, 'removable')
+        reject_device('ro', 1, 'read-only')
+        reject_device('exclusive_access', 0, 'exclusive read-write access')
+        reject_device('path', '/dev/sr0', 'sr0 device')
+        reject_device('path', '/dev/cdrom', 'cdrom device')
+
+        if len(self._rejected_reasons) == 0:
```

Comment:
this can be:

    self._valid = len(self._rejected_reasons) == 0
    return self._valid

Reply:
Thx for improving my style ;)